build/packaging/linux/bin/kend

```diff
@@ -45,12 +45,7 @@ __kill_timeout() {
     kill -9 $PIDNUM && echo "Killed"
 }
 
-start() {
-    __pid_run
-    [ -n "$pid" ] && echo "kend already running...[$pid]" && return
-
-    echo -n "Starting kend: "
-
+__check_option() {
```

Comment:
How about adding this to rpm version as well?

Reply:
@jeongkyun-oh print out rpm version? I think it would be better to proceed with another PR

Reply:
@whoisxx I meant RPM daemon as well, not linux version only. The files are under `build/rpm/etc/init.d/kend`.

Reply:
I will add this to RPM for later.